types/emoji-mart/index.d.ts

```diff
@@ -1,9 +1,10 @@
-// Type definitions for emoji-mart 2.11
+// Type definitions for emoji-mart 3.0.0
```

Comment:
.d.ts files are do not support versioning by patch here, because patches should not introduce changes the the API surface.
```suggestion
// Type definitions for emoji-mart 3.0
```